composer.json

```diff
@@ -7,7 +7,8 @@
         "symfony/finder": "^6.1",
         "symfony/yaml": "^6.1",
         "marcj/topsort": "^2.0.0",
-        "psr/simple-cache": "^3.0.0"
+        "psr/simple-cache": "^3.0.0",
+        "silverstripe/framework": "^5.4.0"
```

Comment:
There was actually already was a [dependency](https://github.com/silverstripe/silverstripe-config/blob/2/src/Collections/MemoryConfigCollection.php#L8) on silverstripe/framework, it just wasn't in composer.json

Reply:
Updated